docs/concepts/workloads/controllers/daemonset.md

```diff
@@ -98,15 +98,41 @@ If you do not specify either, then the DaemonSet controller will create Pods on
 
 ## How Daemon Pods are Scheduled
 
-Normally, the machine that a Pod runs on is selected by the Kubernetes scheduler.  However, Pods
+### Scheduled by DaemonSet controller (default)
+
+Normally, the machine that a Pod runs on is selected by the Kubernetes scheduler. However, Pods
 created by the DaemonSet controller have the machine already selected (`.spec.nodeName` is specified
-when the Pod is created, so it is ignored by the scheduler).  Therefore:
+when the Pod is created, so it is ignored by the scheduler). Therefore:
 
  - The [`unschedulable`](/docs/admin/node/#manual-node-administration) field of a node is not respected
    by the DaemonSet controller.
  - The DaemonSet controller can make Pods even when the scheduler has not been started, which can help cluster
    bootstrap.
 
+### Scheduled by default scheduler (with ScheduleDaemonSetPods alpha feature)
+
+In version 1.10, `ScheduleDaemonSetPods` was introduced as an alpha feature. This features schedules
+ DaemonSet Pods with the default scheduler instead of with the Daemonset controller.
+
+When this feature is enabled, the `NodeAffinity` term is added to the DaemonSet Pods, as in the following example.
+This enables the default scheduler to bind the Pod to the target host.
```

Comment:
Would you also mention that `.spec.nodeName` is not specified? e.g., 
```
... the `NodeAffinity` term (instead of `.spec.nodeName`) is added to the DaemonSet Pods, ...
```